.ci/azure/windows_conditional_compilation.yml

```diff
@@ -60,6 +63,15 @@ jobs:
     SETUPVARS: $(INSTALL_DIR)\setupvars.bat
 
   steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.10.7'
```

Comment:
I suppose we don't need to specify exact version, `major.minor` format is acceptable.
Otherwise we have warnings https://dev.azure.com/openvinoci/dldt/_build/results?buildId=464689&view=results

Reply:
Good point, changed.